advanced-usage/index.md

```diff
@@ -54,11 +54,50 @@ The current time and date is displayed in the top right corner.
 
 ## Edit Mode
 
-Cockpit's interface is designed around a configurable widget system, within an arbitrary set of views.
-Widgets can be added/removed, placed in arbitrary locations, and resized.
+Cockpit's interface consists of a configurable widget system, with
+1. [Profiles](#profiles)
+    - for supporting different operators and/or vehicle types
+    - can be added/removed, saved and loaded, and switched between in edit mode
+1. [Views](#views) (within each profile)
+    - for handling different operation modes / targets within a mission
+    - can be added/removed, saved and loaded, and dynamically switched between
+1. [Widgets](#widgets) (within each view)
+    - for advanced information display and vehicle control
+    - can be added/removed, placed in arbitrary locations, and resized
+1. [Mini-widgets](#mini-widgets) (within mini-widget bar widgets)
+    - for basic information display and vehicle/interface control
+    - can be added/removed, reordered within widget bars, and moved between them
 
 {{ easy_image(src="edit-mode", width=650, center=true) }}
 
+### Profiles
+
+A "profile" is a collection of [views](#views) that are relevant to a particular use-case or operator.
+
+If one control station computer is used by multiple operators (e.g. within the same organisation)
+at different times then it could be useful for each operator to have their preferred interface saved on
+that computer, and they can switch to their profile when they open up Cockpit.
+
+Alternatively, if the same control station computer is used to control different types of vehicles
+(e.g. a boat, an underwater ROV, and a drone) then the operator can load the appropriate vehicle control
+interface when they connect to a different vehicle type. It will soon be possible to store and load
+profiles from the vehicle itself, instead of only on the control station computer, which makes it easier
+to connect a different control computer to a vehicle and load the familiar control profiles for that vehicle.
+
+#### Default Profiles
+
+Cockpit includes default profiles for submarine and boat use-cases, which cannot be edited. These are useful
+for restoring to a known reasonable interface in case something goes wrong with custom interface options.
```

Comment:
[...] as well as getting a reference on how a Profile can look like.
Those default profiles are not persistent, so if they are updated between cockpit versions, they will be updated for you as well.